README.md

```diff
@@ -365,7 +373,25 @@ maintenance and customization.
 
  * `Kuiper with exported sources:` 
 
-   - **07.export-stage** - substage **02.export-sources**
+   - **08.export-stage** - substage **02.export-sources**
+
+ * `Kuiper with custom script:`
+
+   - **07.extra-scripts**
+
+# Kuiper with custom extra scripts
+
+Kuiper has an option to run extra scripts at build time in order to customize the resulted image.
+
+Steps to prepare Kuiper customization with the example script:
+- in the `config` file set 'EXTRA_SCRIPT' variable to `stages/07.extra-scripts/examples/extra-script-example.sh`
```

Comment:
Please specify that the path and script name from here is just an example - the current implementation accepts any path from 'adi-kuiper-gen' directory and any script name. Specify also that supported scripts now are just the bash ones (.sh extension) and that it needs to be executable (+x option). 
Or are you make it executable if it isn't, before running it?